src/assets/data/localisations.ts

```diff
@@ -991,9 +991,19 @@ export const Localisations = {
     es: 'Cargando',
     it: 'Caricamento',
     nl: 'Laden',
-    pl: 'Ładowanie…',
+    pl: 'Ładowanie',
     hb: 'טוען'
   },
+  STATUS_SENDING: {
```

Comment:
Change to `Please wait..`

Reply:
We can also show realtime progress in this like `Please wait... processing 12/250` or similar.